docs/toc.md

```diff
@@ -1,164 +1,93 @@
-# [Getting started](getting-started/index.md)
-## [What is the .NET Core Platform?](getting-started/what-is-dotnet.md)
-## [Installing .NET Core](getting-started/installing/index.md)
-### [.NET Core System Requirements](getting-started/installing/system-reqs.md)
-### [Installing .NET Core on Windows](getting-started/installing/installing-core-windows.md)
-### [Installing .NET Core on Linux](getting-started/installing/installing-core-linux.md)
-### [Installing .NET Core on OS X](getting-started/installing/installing-core-osx.md)
-## [Using the CLI tools to write console apps](getting-started/cli-console-app-tutorial.md)
-## [Setting up your integrated development environment](getting-started/devenv/index.md)
-### [🔧 Development using Visual Studio 2015](getting-started/devenv/using-visual-studio.md)
-### [🔧 Development using Visual Studio Code](getting-started/devenv/using-visual-studio-code.md)
-### [🔧 Development using vim](getting-started/devenv/using-vim.md)
-### [🔧 Development using Sublime Text](getting-started/devenv/using-sublime.md)
+# [About .NET](about/index.md)
+## [.NET Products](about/products.md)
+## [Community](about/community.md)
+## [Open Source](about/open-source.md)
+## [Learning Resources](about/learn.md)
```

Comment:
Learning resources seems a broad enough category that we could include "talks" in it rather than making a separate "talks" doc.